app/controllers/sessions_controller.rb

```diff
@@ -21,6 +21,11 @@ def create
     end
   end
 
+  def destroy
+    session[:current_user_id] = nil
```

Comment:
More readable, and can you add check that unlogging user can't destroy session?
```suggestion
    session.delete(:current_user_id)
```

Reply:
For merge it)

Reply:
Okay👌🏻

Reply:
RuboCop: decorator is missing around sentence [I18n/RailsI18n/DecorateString] what is this?
![image](https://github.com/fs/task_tracker_itis_2023/assets/50077707/22f142c3-0e22-4373-b795-c354774d9ee0)

Reply:
robocop -a did not help

Reply:
You can disable this cop here, it's not important check